### ISSUE TITLE
Installation fails on Linux aarch64 as expected as there is no Bloomberg API support

### ISSUE THREAD

**Issue by pat-s** [Rblp/Rblpapi#399]
```sh
docker run --rm -it --platform linux/arm64 ubuntu:noble bash

apt update && apt install -y r-base g++ cmake
R -q -e 'install.packages("Rblpapi")'

make: *** [Makefile:9: Rblpapi.ts] Error 1
make: Target 'all' not remade because of errors.
* installing *source* package 'Rblpapi' ...
** package 'Rblpapi' successfully unpacked and MD5 sums checked
** using staged installation
Unknown architecture: aarch64. Exiting.
ERROR: configuration failed for package 'Rblpapi'
```

**Comment by eddelbuettel** [comment]
Thanks for testing, and very timely as we are nearing a release.  _We know it works on non-cross-compile arm64_ macOS (as seen on both mac-builder and GHA using arm64 natively) so can  you please clarify for us non-macOS users what you are doing?  I.e.

- what is your host os, and what is your guess os?
- are you running natively or in some type of emulation?
- please run `./configure` directly and report that
- please look into what `configure` does -- chiefly `uname -m` -- and check whether is right or wrong

In short we need you to debug this as we a) do not have the platform at hand and b) have it working native on arm64.

**Comment by eddelbuettel** [comment]
I will expand DESCRIPTION::Description in the upcoming release with the added

> Currently supported platforms are Windows and Linux (both for
 x86-64 processors only) as well as macOS (for arm64 only).  Other os-cpu pairings
 are not supported by Bloomberg and can therefore not be supported here.

which makes it more explicit up-front too.
